src/idl/rrdb.thrift

```diff
@@ -253,6 +253,7 @@ struct get_scanner_request
     9:filter_type  sort_key_filter_type;
     10:dsn.blob    sort_key_filter_pattern;
     11:optional bool    validate_partition_hash;
+    12:optional bool    return_expire_ts;
```

Comment:
Would it better to add `return_expire_ts` to `struct key_value`, like:
```
struct key_value
{
    1:dsn.blob      key;
    2:dsn.blob      value;
    3:optional bool expire_ts;
}
```
It would be more general and extensible.

Reply:
Good idea ! expire_ts should be a member of key_value.  Should we use `i32` instead of `bool` for `expire_ts` which is used in `scan_response` to return all of the key-value pairs with optional expire_ts:
```
struct key_value
{
    1:dsn.blob      key;
    2:dsn.blob      value;
    3:optional i32 expire_ts;
}
```
where I've used an `expire_ts_seconds_list` to separate the optional `expire_ts` to another list. However, since `return_expire_ts` in get_scanner_request tells the server that the client need expire_ts, should we keep it ?

Reply:
Oh, yes, it's type of `i32`, and you can keep it in `get_scanner_request`.

Reply:
OK

Reply:
To follow the previous style, `expire_ts` is fixed as `expire_ts_seconds`.